src/promptflow/promptflow/_sdk/_submitter/utils.py

```diff
@@ -313,6 +313,16 @@ def print_chat_output(output, generator_record):
         print(output)
 
 
+def print_csharp_stream_chat_output(output, chat_output_name):
```

Comment:
why specially say csharp here? it should be the same if any lang that using executor service?

Reply:
maybe we can revisit this later

Reply:
offline synced